schema.graphql

```diff
@@ -33,3 +32,82 @@ type AuctionedName @entity {
   bidCount: Int!
   state: AuctionState
 }
+
+type Resolver @entity {
+  id: ID!
+  domain: Bytes!
+  resolverEvents: [ResolverEvent!]! @derivedFrom(field: "resolver")
+}
+
+interface ResolverEvent {
+  id: ID!                   # Count events in order - 0,1,2,3 etc
```

Comment:
Couldn't we use block number / log ID here, in order to avoid the need to keep separate counters?

Reply:
Yes, this is a more elegant pattern to use. I will change it